nirum_wsgi.py

```diff
@@ -559,9 +559,10 @@ def __bool__(self):
     __nonzero__ = __bool__
 
     def update(self, match_result):
-        self.result = List(
-            itertools.chain(self.result, match_result.result)
-        )
+        if self.result or match_result:
```

Comment:
I think it would be better initialize `self.result` with an empty list(`List()`) on `__init__`. So we can only check `match_result` & `match_result.result` isn't `None`.

```
if match_result and match_result.result:
    self.result = List(itertools.chain(self.result, match_result.result)
```

Reply:
Umm,,, I tried like that, but other methods using this class have different behavior when `self.result` is `None` or `List()`.

Reply:
Maybe you can change `__bool__` like below:

```
def __bool__(self):
    return bool(self.value)
```

Reply:
Due to exact match of path, we should separate initialization with `[]` and `None`